Assets/MixedRealityToolkit/_Core/Definitions/Devices/SupportedControllerType.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices
+{
+    /// <summary>
+    /// The SDKType lists the XR SDKs that are supported by the Mixed Reality Toolkit.
+    /// Initially, this lists proposed SDKs, not all may be implemented at this time (please see ReleaseNotes for more details)
```

Comment:
Why not just add what we've implemented?

Reply:
Can you elaborate?  We do not have a distinct list of "supported" controllers anywhere else.  Can't use SystemType as we can't enumerate them (other than in the inspector)

Reply:
I'm just trying to say, why not just add the ones we've implemented instead of asking users to check the release notes. 

Only add a controller type if it's been implemented.

Reply:
These are the only one's I've implemented